ts/components/LabelledItem.tsx

```diff
@@ -128,14 +136,18 @@ export class LabelledItem extends React.Component<Props, State> {
             this.props.isValid === undefined ? false : this.props.isValid
           }
         >
-          {this.props.icon && (
-            <IconFont
-              size={variables.iconSize3}
-              color={variables.brandDarkGray}
-              name={this.props.icon}
-              style={this.props.iconStyle}
-            />
-          )}
+          {this.props.icon &&
+            (isString(this.props.icon) ? (
+              <IconFont
+                size={variables.iconSize3}
+                color={variables.brandDarkGray}
+                name={this.props.icon}
+                style={this.props.iconStyle}
+              />
+            ) : (
+              <Image source={this.props.icon} style={[this.props.iconStyle]} />
```

Comment:
```suggestion
              <Image source={this.props.icon} style={this.props.iconStyle} />
```
It should not be needed

Reply:
OK for square brackets. I configured prettier to run on save only in the io-app-cie workspace :-)